completions/bash/ocitools

```diff
@@ -209,6 +209,8 @@ _ocitools_ocitools() {
 	local boolean_options="
 		$global_boolean_options
 		--help
+		--host-specific
+		--log-level
```

Comment:
On Tue, Aug 16, 2016 at 08:54:35PM -0700, Ma Shimiao wrote:
> > +		--log-level
> 
> On 08/17/2016 11:38 AM, W. Trevor King wrote:
> > This is not a boolean option
> > <https://github.com/opencontainers/ocitools/blob/6aeb752f0c9c4142a50b00d872b2b6d0c400dd0f/cmd/ocitools/main.go#L16-L20>
>
> fixed.

Want to update the man page with =LEVEL while we're at it?